Update active-directory-conditional-access-policy-connected-applicati…

JairoACadena opened the pull request:
…ons.md

Removing 3rd party MDMs from compliance. Marking compliant bit in Azure AD is a EMS reserved value proposition.

azurecla commented:
Hi __@alex-frankel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

tysonn commented:
The images don't appear in the GitHub rendered view, so it seems like something is broken. Please check the links, filenames, capitalization of links and file names to ensure capitalization matches.

JairoACadena committed:
Update active-directory-conditional-access-policy-connected-applications.md

Removing 3rd party MDMs from compliance. Marking compliant bit in Azure AD is a EMS reserved value proposition.
